src/test/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTaskTest.java

```diff
@@ -85,6 +85,11 @@
     private static final String NOT_EXISTING_FILE = "target/not_existing.xml";
     private static final String FAILURE_PROPERTY_VALUE = "myValue";
 
+    @Override
+    protected String getPackageLocation() throws IOException {
+        return "com/puppycrawl/tools/checkstyle";
```

Comment:
We already have an `checkstyleanttask` folder inside `ant`, so this package location should be: `com/puppycrawl/tools/checkstyle/ant/checkstyleanttask`.